salt/metalk8s/kubernetes/mark-control-plane/files/bootstrap_node_update.yaml.j2.in

```diff
@@ -14,6 +14,14 @@
 {%- if 'node-role.kubernetes.io/node' not in node_obj['metadata'].get('labels', {}) %}
   {%- set node_taints = node_obj['spec'].get('taints') | default([], True) %}
 
+  {#- Convert all `time_added` key to string as datetime cannot be serialized
```

Comment:
Can't we add this at Jinja's level ? I mean, overloading the `tojson` filter to handle datetime objects (maybe it does not worth it, just to know if it's doable), so we'll handle it for any other object we would need to pass into this filter.

Reply:
Hmm I think it's doable with salt, but I never tested.
I think it's not worth for the moment but maybe it will be in the future, good idea

Reply:
FWIW, using `| yaml` instead of `| tojson`, which is preinstalled with Salt, handles date objects just fine :wink: